tests/Validation/ValidationValidatorTest.php

```diff
@@ -14,6 +14,21 @@ public function tearDown()
         m::close();
     }
 
+    public function testCanExtractRuleAndParametersFromARuleObject()
+    {
+        $rule = m::mock(\Illuminate\Validation\Rules\Rule::class);
+
+        // Why 6 times?
+        $rule->shouldReceive('toArray')->times(6)->andReturn(['max', [120]]);
```

Comment:
I wanted to make sure "toArray" was being called instead of "__toString". I then found out that __toString was being called 12 times! And toArray 6 (something that I might like to fix).